### ISSUE TITLE
VIP not allowed to move, even when freed

### ISSUE THREAD

**Issue by StuartA101** [long-war-2/lwotc#1121]
On a rescue VIP mission, even though all enemies are vanquished, the door of the van has been hacked & opened the VIP doesn't have any movement options, he can only hunker down.

The only way to finish the level is to kill him with a grenade.

This has happened twice now (same/similar map too).

Save Game file attached.

[save_No_Chance_Of_Rescue.zip](https://github.com/long-war-2/lwotc/files/5232085/save_No_Chance_Of_Rescue.zip)

**Comment by dpstafford** [comment]
I have found that sometimes you have to send a trooper inside the van.   Maybe to make visual contact.  


    On Wednesday, September 16, 2020, 08:21:58 AM EDT, StuartA101 <notifications@github.com> wrote:  
 
 


On a rescue VIP mission, even though all enemies are vanquished, the door of the van has been hacked & opened the VIP doesn't have any movement options, he can only hunker down.

The only way to finish the level is to kill him with a grenade.

This has happened twice now (same/similar map too).

Save Game file attached.

save_No_Chance_Of_Rescue.zip

**Comment by StuartA101** [comment]
> I have found that sometimes you have to send a trooper inside the van.   Maybe to make visual contact.   

It won't allow a trooper to go into the van (or I would of tried KO'ing him and carrying him), its like the floor of the van won't allow any movement at all

**Comment by dpstafford** [comment]
Did you open the door after hacking it?Are your men blocking the exit?On Sep 16, 2020 9:08 AM, StuartA101 <notifications@github.com> wrote:



I have found that sometimes you have to send a trooper inside the van.   Maybe to make visual contact.  



It won't allow a trooper to go into the van (or I would of tried KO'ing him and carrying him), its like the floor of the van won't allow any movement at all

**Comment by StuartA101** [comment]
> Did you open the door after hacking it?Are your men blocking the exit?

Yes, of course. Hacked door, opened door with a trooper, made visual contact... but thats all that it allows!

**Comment by dpstafford** [comment]
Maybe it is a mod conflict.   As I have never encounter this after countless VIP rescues in LWOTC.



    On Thursday, September 17, 2020, 04:09:56 AM EDT, StuartA101 <notifications@github.com> wrote:  
 
 



Did you open the door after hacking it?Are your men blocking the exit?


Yes, of course. Hacked door, opened door with a trooper, made visual contact... but thats all that it allows!

**Comment by StuartA101** [comment]
Hmmm... had a look through my 'running' mods. I do have a map mod running, so maybe it is that.
Ironically, I've been very cautious with Mods with LWOTC, as one conflicted early on, so I turned them all off and slowly slowly added more in and must have hit the map mod by accident.

Not sure if that is the conflict, as all maps seem to integrate well with the game, its just that minor thing.. which I`ll take off next run through.

The map is the same one - its like a big dam or reservoir (modern setting, built up) - not sure if thats an original map or one from the mod TBH.

**Comment by StuartA101** [comment]
I started a new game (as I lost lol).. no map mods running... pretty far in and came across the same issue. See pic.
![LWOTC_No_escape](https://user-images.githubusercontent.com/69280141/97373134-2428e200-18ad-11eb-83dd-799b590356af.jpg)

**Comment by dpstafford** [comment]
I don't see a friendly soldier standing next to the VIP.   As I said, I run in to this all the time, and I have to send someone inside the truck to activate the VIP.  One could argue that shouldn't be necessary.  But.
I never have any trouble sending a trooper into the truck and the VIP always releases when I do.   Beta 3.1.

**Comment by StuartA101** [comment]
On other map/missions, the VIP is able to move once the doors are open and he is sighted.
On this one, it wont allow any troopers into the van.. but I`ll try it again next time it comes up.
It seems to be this one map, which I thought was part of a map mod, but apparently isnt.
